skt/reporter.py

```diff
@@ -297,27 +297,26 @@ def __getjobresults(self):
                 task_node = self.__get_task(recipe, test_name)
                 if task_node.attrib['result'] != 'Pass':
                     failed_tasks.append(task_node)
-                    continue
```

Comment:
I thought this `continue` was weird while reviewing the original patch, but didn't have the capacity to review this properly. Sigh...